utils/github_utils.py

```diff
@@ -97,6 +97,7 @@ def release_notes(pull_request_number: int) -> NoReturn:
         "feature": "",
         "techdebt": "",
         "tests": "",
+        "improvement": "",
```

Comment:
Isn't it very similar to enhancement?

Reply:
Yes, I don't know what was marked as improvement, but without it in the map, it breaks the util.